node/config.go

```diff
@@ -502,8 +502,13 @@ func (c *Config) warnOnce(w *bool, format string, args ...interface{}) {
 }
 
 type LogConfig struct {
-	FileRoot     string
-	FilePath     string
-	MaxBytesSize uint
-	Level        string
+	FileRoot     *string
```

Comment:
why change these value to pointer? for efficiency？
if true, I think there is no need to do so,
because they are not on critical path for efficiency